app/src/main/java/com/lastaoutdoor/lasta/data/auth/AuthRepositoryImpl.kt

```diff
@@ -98,6 +106,11 @@ constructor(
       emit(Response.Loading)
       val user = auth.currentUser
       if (user != null) {
+        userDBRepo.deleteUser(user.uid)
```

Comment:
I really like the fact that you put everything in the auth repo, makes it real consise!